base/intfuncs.jl

```diff
@@ -743,12 +743,25 @@ julia> digits!([2,2,2,2,2,2], 10, base = 2)
  0
 ```
 """
-function digits!(a::AbstractVector{T}, n::Integer; base::Integer = 10) where T<:Integer
-    base < 0 && isa(n, Unsigned) && return digits!(a, convert(Signed, n), base = base)
+function digits!(a::AbstractVector{T}, n::Integer; base::Integer = 10, skipfirst=false) where T<:Integer
```

Comment:
Should we call this `__skipfirst__` or something to make it 100% clear that it's not part of the public API?

Reply:
I had only thought about `_skipfirst` ! and then didn't bother to re-push, as it's not in the docstring and it's not possible to use this keyword by accident (like it would be with a positional with default). But I realize now that i appears in the output of `methods(digits!)`...  I guess I should rename.

Reply:
Gripe: if we had an in-place `circshift!(a, 1)`, this keyword wouldn't be needed.

Reply:
`_skipfirst` would be fine too. Depends on how many underscores you need to feel safe.